sky/packages/sky/lib/src/fn3/focus.dart

```diff
@@ -137,8 +140,9 @@ class FocusState extends State<Focus> {
   void _scopeRemoved(GlobalKey key) {
     assert(_focusedScope == key);
     _currentlyRegisteredScopeRemovalListenerKey = null;
-    setState(() {
-      _focusedScope = null;
+    _focusedScope = null;
```

Comment:
We won't update this frame because we send the notifications after we're done calling rebuild.